backend/transcribee_backend/metrics.py

```diff
@@ -0,0 +1,128 @@
+import datetime
+import secrets
+from abc import abstractmethod
+from typing import List
+
+from fastapi import Depends, HTTPException, status
+from fastapi.security import HTTPBasic, HTTPBasicCredentials
+from prometheus_client import Gauge
+from sqlmodel import Session, col, func
+
+from transcribee_backend.config import settings
+from transcribee_backend.db import SessionContextManager
+from transcribee_backend.helpers.time import now_tz_aware
+from transcribee_backend.models.document import Document
+from transcribee_backend.models.task import Task, TaskAttempt, TaskState
+from transcribee_backend.models.user import User
+from transcribee_backend.models.worker import Worker
+
+
+class Metric:
+    @abstractmethod
+    def refresh(self, session: Session):
+        pass
+
+
+class TasksInState(Metric):
```

Comment:
I would rather like to have this as TaskChanis or TaskGraphs in state. Otherwise this number is not really saying much (Transcode is much faster, yadayada)

Reply:
I would prefer some other task parameters (that influence needed computing time) as labels then too